climada/hazard/low_flow.py

```diff
@@ -571,39 +708,120 @@ def _read_single_nc(filename, yearrange, bbox):
             data = data.sel(time=slice(cftime.DatetimeNoLeap(yearrange[0], 1, 1),
                                        cftime.DatetimeNoLeap(yearrange[-1], 12, 31)))
         else:
-            data = data.sel(lon=slice(bbox[0], bbox[2]), lat=slice(bbox[3], bbox[1]),
+            lon_min, lat_min, lon_max, lat_max = bbox
+            data = data.sel(lon=slice(lon_min, lon_max), lat=slice(lat_max, lat_min),
                             time=slice(cftime.DatetimeNoLeap(yearrange[0], 1, 1),
                                        cftime.DatetimeNoLeap(yearrange[-1], 12, 31)))
         datetimeindex = data.indexes['time'].to_datetimeindex()
         data['time'] = datetimeindex
     return data
 
 
+def _xarray_reduce(data, fun=None, percentile=None):
```

Comment:
Could this function be a future canditate for the climada.util module or is it too specific?

Reply:
i guess it is a bit too specific